purchase_request/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+{
+    "name": "Purchase Request",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "version": "11.0.1.0.2",
+    "summary": "Use this module to have notification of requirements of "
+               "materials and/or external services and keep track of such "
+               "requirements.",
+    "category": "Purchase Management",
+    "depends": [
+        "purchase",
+        "product"
+    ],
+    "data": [
```

Comment:
@hveficent IMO `data` sequence like [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)